ortho_seq_code/gui/gui.py

```diff
@@ -74,14 +76,16 @@ def __init__(self, *args, **kwargs):
         self.widget_layout.addWidget(self.poly_order_combobox)
         self.widget_layout.addLayout(upload_ComboBox2_layout)
 
-        # Precomputed checkboxes
+        # precomputed dir path
         upload_layout3 = QHBoxLayout()
-        label4 = QLabel("Precomputed?")
+        label4 = QLabel("Select directory with precomputed sequence file:")
         upload_layout3.addWidget(label4)
-        self.precomputed_combobox = QComboBox()
-        self.precomputed_combobox.addItems(["No", "Yes"])
-        upload_layout3.addWidget(self.precomputed_combobox)
-        self.widget_layout.addLayout(upload_layout3)  # for checkbox
+        self.upload_button_3 = QPushButton("precomputed_dir")
```

Comment:
I think a name like `upload_precomputed_dir_button` is more descriptive.

Reply:
cool, done!